tests/integration/nm/ovs_test.py

```diff
@@ -229,102 +226,6 @@ def _get_parent_uuid_of_interface(iface_name):
-            assertlib.assert_state_match(state)
-            new_state = statelib.show_only((BRIDGE0, IFACE0))
-            assert (
-                new_state[Interface.KEY][0][OvsDB.OVS_DB_SUBTREE][
-                    OvsDB.EXTERNAL_IDS
-                ]
-                == OVSDB_EXT_IDS_CONF1_STR
-            )
-            assert (
-                new_state[Interface.KEY][1][OvsDB.OVS_DB_SUBTREE][
-                    OvsDB.EXTERNAL_IDS
-                ]
-                == OVSDB_EXT_IDS_CONF2_STR
-            )
-
-    def test_modify_ovs_bridge_external_ids(self, disable_ovsdb_plugin):
-        bridge = Bridge(BRIDGE0)
-        bridge.set_ovs_db({OvsDB.EXTERNAL_IDS: OVSDB_EXT_IDS_CONF1})
-        bridge.add_internal_port(
-            IFACE0,
-            ipv4_state={InterfaceIPv4.ENABLED: False},
-            ovs_db={OvsDB.EXTERNAL_IDS: OVSDB_EXT_IDS_CONF2},
-        )
-        with bridge.create():
-            changed_bridge = Bridge(BRIDGE0)
-            changed_bridge.set_ovs_db(
-                {OvsDB.EXTERNAL_IDS: OVSDB_EXT_IDS_CONF2}
-            )
-            changed_bridge.add_internal_port(
-                IFACE0,
-                ipv4_state={InterfaceIPv4.ENABLED: False},
-                ovs_db={OvsDB.EXTERNAL_IDS: OVSDB_EXT_IDS_CONF1},
-            )
-            changed_bridge.apply()
-            assertlib.assert_state_match(changed_bridge.state)
-            new_state = statelib.show_only((BRIDGE0, IFACE0))
-            assert (
-                new_state[Interface.KEY][0][OvsDB.OVS_DB_SUBTREE][
-                    OvsDB.EXTERNAL_IDS
-                ]
-                == OVSDB_EXT_IDS_CONF2_STR
-            )
-            assert (
-                new_state[Interface.KEY][1][OvsDB.OVS_DB_SUBTREE][
-                    OvsDB.EXTERNAL_IDS
-                ]
-                == OVSDB_EXT_IDS_CONF1_STR
-            )
-
-    def test_ovs_bridge_remoev_external_ids(self, disable_ovsdb_plugin):
-        bridge = Bridge(BRIDGE0)
-        bridge.set_ovs_db({OvsDB.EXTERNAL_IDS: OVSDB_EXT_IDS_CONF1})
-        bridge.add_internal_port(
-            IFACE0,
-            ipv4_state={InterfaceIPv4.ENABLED: False},
-            ovs_db={OvsDB.EXTERNAL_IDS: OVSDB_EXT_IDS_CONF2},
-        )
-        with bridge.create():
-            changed_bridge = Bridge(BRIDGE0)
-            changed_bridge.set_ovs_db({OvsDB.EXTERNAL_IDS: {}})
-            changed_bridge.add_internal_port(
-                IFACE0,
-                ipv4_state={InterfaceIPv4.ENABLED: False},
-                ovs_db={OvsDB.EXTERNAL_IDS: {}},
-            )
-            changed_bridge.apply()
-            assertlib.assert_state_match(changed_bridge.state)
-            new_state = statelib.show_only((BRIDGE0, IFACE0))
-            assert not new_state[Interface.KEY][0][OvsDB.OVS_DB_SUBTREE][
-                OvsDB.EXTERNAL_IDS
-            ]
-            assert not new_state[Interface.KEY][1][OvsDB.OVS_DB_SUBTREE][
-                OvsDB.EXTERNAL_IDS
-            ]
```

Comment:
Why are these tests being removed?

Reply:
Because there is no overlap beetwen ovsdb plugin and NM  plugin, the ovsdb plugin is only for global configure while NM plugin for interface level configure. Current existing test case already has ovsdb tests.
Those deleted test case is testing whether NM plugin can change interface level ovsdb, so not required anymore.